modules/minigl/src/test/java/org/jpos/gl/stress/CreateTransactions.java

```diff
@@ -23,15 +23,18 @@
 import java.util.Date;
 import java.util.Random;
 import org.hibernate.Transaction;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
 
 public class CreateTransactions extends TestBase {
     Random r = new Random(0); // we wan't a consistent set 
     Date POSTDATE;
     public static final BigDecimal ZERO = new BigDecimal ("0.00");
+    @BeforeEach
     public void setUp () throws Exception {
-        super.setUp();
```

Comment:
why do you delete calls to `super.setUp()` where the proper GLSession (with the right name for the tests) get initialized?

Reply:
Isn't that unnecessary due to the `TestBase` changes?